README.md

```diff
@@ -79,7 +79,7 @@ The prediction API code can be found at the [repo](https://github.com/deven96/Im
 
 The `master` branch of the repo is linked to automatically [deployed](https://bisoncorps-imageq.herokuapp.com)
 ![Homepage](docs/assets/homepage.png)
- and it sends requests to the deployed [prediction API](https://imageqapi.appspot.com/)
+and it sends requests to the deployed [prediction API](https://imageqapi.appspot.com/)
```

Comment:
We need to add a `pylint` and `autopep8` configuration that handles things like white space and setup a commit hook that auto formats the code before pushing. 
I'll create an issue and self assign it